js/client-mainmenu.js

```diff
@@ -114,7 +126,131 @@
 				);
 			}
 		},
+		createLoginFormObserver:function(self){
+			const config = { attributes: true, childList: true, subtree: true };
+			const observer = new MutationObserver(function(){
+			if(window.app.user.get("named") && window.Storage.prefs("page")=="challenge"){
+				window.Storage.prefs("page","testclient");
+				if($('.challenge-container').length==0){
+					//user log in event trigerd when user log in and make user pick a team
+					$('body').on("login_event",function(){
+					self.onUserLogIn(self);
+				})
+				$('body').trigger("login_event");
+				}
+			}
+			});
+			observer.observe(document.activeElement, config);
+			},
+		onUserLogIn:function(self){
```

Comment:
Run eslint please.

Reply:
ther was issue when runing eslint it adds taps and spaces , and it also change strings to template strings

Reply:
> ther was issue when runing eslint it adds taps and spaces

Yes, that's proper behavior. It's supposed to do that to help format your code properly.

> t also change strings to template strings

I'd advise removing them after running eslint manually.